### PR TITLE
vendor: containerd/containerd v1.4.1

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -10,7 +10,7 @@ github.com/BurntSushi/toml                          v0.3.1
 github.com/cespare/xxhash/v2                        v2.1.1
 github.com/containerd/cgroups                       318312a373405e5e91134d8063d04d59768a1bff
 github.com/containerd/console                       v1.0.0
-github.com/containerd/containerd                    v1.4.0
+github.com/containerd/containerd                    v1.4.1
```
Comment (mikebrow) [inline vendor.conf:13]: should we just move cri/master to containerd master or stage it up to 1.4.1  

if the former need to grab: 

```
github.com/containerd/cgroups                       0b889c03f102012f1d93a97ddd3ef71cd6f4f510
github.com/containerd/ttrpc                         v1.0.2
```

fair to do both.. this first then master

**Review by Zyqsempai — APPROVED** [review]
LGTM

**Review by mikebrow — APPROVED** [review]
LGTM
see question..

File: vendor.conf (hunk 1 above)
Comment (thaJeztah) [reply vendor.conf:13]: Ah, yes, I was considering wether to do this on master, then cherry-pick to release/1.4, or directly on the 1.4 branch. guess either could work.

IIUC, "soon" master branch will no longer be a thing (as in; merged into containerd/containerd 🤞)

Comment (mikebrow) [reply vendor.conf:13]: kk will merge this .. first allowing a cherry.. so much fun... then we can consider if we want to push a release for cri/ (the last one really) to 1.19.1

**Pull request merged**